### PR TITLE
Added recipe for installation of LatLon23 library

### PR DESCRIPTION
**Pull request by jgliss**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jgliss** [comment]
I am willing to be listed here.

File: recipes/latlon23/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e0ea06eda0ac1dab658fec89ee158c5221948cf147e38724fe50dfbd8a1ff340
+
+build:
+  number: 0
```
Comment (chrisburr) [inline recipes/latlon23/meta.yaml]:
```suggestion
  number: 0
  noarch: python
```

File: recipes/latlon23/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - pyproj
```
Comment (chrisburr) [inline recipes/latlon23/meta.yaml]: I suspect this isn't needed in `host`
```suggestion
```

File: recipes/latlon23/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  imports:
+    - LatLon23
+
+about:
+  home: https://github.com/hickeroar/LatLon23
+  license: GNU General Public v3 (GPLv3)
+  license_family: GPL3
+  license_file: LICENSE
+  summary: Methods for representing geographic coordinates
+  doc_url:
```
Comment (chrisburr) [inline recipes/latlon23/meta.yaml]: If there isn't a URL for documentation this field should just be removed
```suggestion
```

File: recipes/latlon23/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    - pyproj
+    - python
+
+test:
+  imports:
+    - LatLon23
+
+about:
+  home: https://github.com/hickeroar/LatLon23
+  license: GNU General Public v3 (GPLv3)
```
Comment (chrisburr) [inline recipes/latlon23/meta.yaml]: From https://opensource.org/licenses/alphabetical
```suggestion
  license: GPL-3.0
```

File: recipes/latlon23/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  license_family: GPL3
```
Comment (chrisburr) [inline recipes/latlon23/meta.yaml]:
```suggestion
  license_family: GPL
```

**Pull request merged**